x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -158,25 +142,22 @@ export const isolationRequestHandler = function (
       });
     }
 
-    const commentLines: string[] = [];
-
-    commentLines.push(`${isolate ? 'I' : 'Uni'}solate action was sent to the following Agents:`);
-    // lines of markdown links, inside a code block
-
-    commentLines.push(`${agentIDs.map((a) => `- [${a}](/app/fleet#/agents/${a})`).join('\n')}`);
-    if (req.body.comment) {
-      commentLines.push(`\n\nWith Comment:\n> ${req.body.comment}`);
-    }
-
     // Update all cases with a comment
     if (caseIDs.length > 0) {
       await Promise.all(
         caseIDs.map((caseId) =>
           casesClient.attachments.add({
             caseId,
             comment: {
-              comment: commentLines.join('\n'),
-              type: CommentType.user,
+              type: CommentType.actions,
+              comment: req.body.comment || '',
+              actions: {
+                targets: agentIDs.map((x: HostMetadata) => ({
```

Comment:
optional suggestion: use a better descriptor for the `x` variable - maybe `metadata`?